tools/vms-generator/utils/utils.go

```diff
@@ -131,28 +131,24 @@ func addRNG(spec *v1.VirtualMachineInstanceSpec) *v1.VirtualMachineInstanceSpec
 }
 
 func addContainerDisk(spec *v1.VirtualMachineInstanceSpec, image string, bus string) *v1.VirtualMachineInstanceSpec {
-	spec.Domain.Devices = v1.Devices{
```

Comment:
how are these changes related to the PR?

Reply:
Look at `GetVMIMigratable`. With the existing `addContainerDisk` it wouldn't work because the interface would be clear by this fucntion. This prevents that unexpected side-effect of `addContainerDisk`